src/main/java/net/modfest/scatteredshards/api/shard/ShardType.java

```diff
@@ -30,12 +30,8 @@ public record ShardType(int textColor, int glowColor, Optional<ParticleType<?>>
 	public static final SoundEvent COLLECT_VISITOR_SOUND = SoundEvent.of(ScatteredShards.id("collect_visitor"));
 	public static final SoundEvent COLLECT_CHALLENGE_SOUND = SoundEvent.of(ScatteredShards.id("collect_challenge"));
 	public static final SoundEvent COLLECT_SECRET_SOUND = SoundEvent.of(ScatteredShards.id("collect_secret"));
-	
-	public static final ShardType VISITOR = new ShardType(0x6DE851, 0x00FF48, Optional.of(ParticleTypes.TOTEM_OF_UNDYING), Optional.of(COLLECT_VISITOR_SOUND));
```

Comment:
These still need to exist in case data is missing, unless you add them back in as data and prefer not-our-data.